docs/index.rst

```diff
@@ -55,7 +55,7 @@ execution environment. You can specify these items:
 Choosing a base image
 ---------------------
 
-You can use any base image you choose.
+You can use any base image you choose, as long as it's a RHEL-type image like CentOS, Fedora, or RHEL. These operating systems are directly supported.
```

Comment:
I think this reads slightly better.

```suggestion
The base image must be a RHEL image or one of the RHEL-based variants (CentOS, Fedora, etc.). Other distributions are not supported at this time.
```

Reply:
I wouldn't say "RHEL-based variants", since CentOS and Fedora are not based on RHEL (as Ubuntu is based on Debian). Maybe say something like "... or one of the similar distributions (CentOS, ..."?